example/rust/src/docs_examples/docs_examples.rs

```diff
@@ -651,9 +656,72 @@ pub async fn example_responsetypes_dictionary_with_try_into(cache_client: &Cache
     Ok(())
 }
 
+#[allow(non_snake_case)]
+pub fn example_API_Storage_InstantiateClient() -> Result<(), MomentoError> {
+    let _storage_client = PreviewStorageClient::builder()
+        .configuration(momento::storage::configurations::Laptop::latest())
+        .credential_provider(CredentialProvider::from_env_var(
+            "MOMENTO_API_KEY".to_string(),
+        )?)
+        .build()?;
+    Ok(())
+}
+
+#[allow(non_snake_case)]
+pub async fn example_API_Storage_CreateStore(storage_client: &PreviewStorageClient, store_name: &String) -> Result<(), MomentoError> {
+    let response = storage_client.create_store(store_name).await?;
+    match response {
+        momento::storage::CreateStoreResponse::Created => println!("Store {} created", store_name),
+        momento::storage::CreateStoreResponse::AlreadyExists => println!("Store {} already exists", store_name),
+    }
+    Ok(())
+}
+
+#[allow(non_snake_case)]
+pub async fn example_API_Storage_ListStores(storage_client: &PreviewStorageClient) -> Result<(), MomentoError> {
+    let response = storage_client.list_stores().await?;
+    println!("Stores: {:#?}", response.stores);
+    Ok(())
+}
+
+#[allow(non_snake_case)]
+pub async fn example_API_Storage_DeleteStore(storage_client: &PreviewStorageClient, store_name: &String) -> Result<(), MomentoError> {
+    storage_client.delete_store(store_name).await?;
+    println!("Store {} deleted", store_name);
+    Ok(())
+}
+
+#[allow(non_snake_case)]
+pub async fn example_API_Storage_Put(storage_client: &PreviewStorageClient, store_name: &String) -> Result<(), MomentoError> {
+    storage_client.put(store_name, "key", "value").await?;
+    println!("Put key and value in store {}", store_name);
+    Ok(())
+}
+
+#[allow(non_snake_case)]
+pub async fn example_API_Storage_Get(storage_client: &PreviewStorageClient, store_name: &String) -> Result<(), MomentoError> {
+    let response = storage_client.get(store_name, "key").await?;
+    match response {
+        momento::storage::GetResponse::NotFound => println!("Key not found in {}", store_name),
+        momento::storage::GetResponse::Found { value } => {
+            let found_value: String = value.try_into()?;
+            println!("Got value {}", found_value);
+        }
+    }
```

Comment:
In the go sdk examples, you have `put` with various types, and a switch over found values to test for each data type. I am not clear what the pattern is for the examples but we should be consistent.